### ISSUE TITLE
Cannot create a new task

### ISSUE THREAD

**Issue by richarddribeiro** [ariejan/txdx#3]
Hi Ariejan,

I have found something that I am not sure if it is caused by something in my system, or it is a bug. Can you have a look, please?

**Describe the bug**
When I click at the "+" sign (top right) in the main window a quick motion appears showing what it should be a window to type the text for a new task, but suddenly it closes and the task is not created.

**To Reproduce**
1. Click at the "+" sign (top right).
2. Look at quick window that appears at the top of the tasks' list.
3. The window closes.
4. The new task is not created.

**Expected behavior**
A window to type the data regarding a new task.

**Screenshots**
Unfortunately the motion that creates the window and closes it is too fast to be captured.

**Desktop (please complete the following information):**
 - OS: Manjaro
 - Version: Manjaro 21.2.6 Qonos

**Additional context**
`.xsession-erros` shows:
`** (txdx:12290): CRITICAL **: 11:11:10.058: void update_pressing_state(FlKeyEmbedderResponder *, uint64_t, uint64_t): assertion 'lookup_hash_table(self->pressing_records, physical_key) == 0' failed`

**Comment by otaj** [comment]
Hi, I'm on Arch, the same version of txdx, tried building from source (`txdx` package in AUR) as well as usiung binary release (`txdx-bin`) and have the same issue. What is interesting is the fact, that it appears to happen only after adding first task.

Thanks!

**Comment by ariejan** [comment]
@otaj @richarddribeiro thanks for reporting this, I'm aware of the issue, it's not Manjaro specific. I'm rolling out 1.1.3 right now, it should be available in about 30 minutes on AUR.
